src/rule-resolver/get-rule-definitions.test.ts

```diff
@@ -1,9 +1,14 @@
-import getRuleDefinitions, { RULE_SUFFIX } from './get-rule-definitions';
-
-jest.mock('path');
-jest.mock('./../utils');
+import { RULE_SUFFIX } from './get-rule-definitions';
 
 describe('getRuleDefinitions', () => {
+    let getRuleDefinitions: any;
+    beforeEach(() => {
+        jest.resetModules();
+        jest.restoreAllMocks();
+        jest.doMock('path');
+        jest.doMock('./../utils');
+        getRuleDefinitions = require('./get-rule-definitions').default;
+    });
```

Comment:
Mocks were leaking between tests so I refactored how we were mocking and importing the unit under test slightly.

Reply:
Thanks for fixing it.